src/sentry/api/helpers/group_index/update.py

```diff
@@ -173,7 +173,7 @@ def get_current_release_version_of_group(
 
 def update_groups(
     request: Request,
-    group_ids: Sequence[Group],
+    group_ids: Sequence[int],
```

Comment:
this was a weird type mismatch. Check how it's used below and change should make sense.